scripts/receive_noaa.sh

```diff
@@ -167,7 +167,10 @@ for enhancement in $ENHANCEMENTS; do
       push_annotation="${push_annotation} Max Elev: ${SAT_MAX_ELEVATION}°"
       push_annotation="${push_annotation} Sun Elevation: ${SUN_ELEV}°"
       push_annotation="${push_annotation} | ${PASS_DIRECTION}"
-
+      
+    filesize=$(wc -c "${IMAGE_FILE_BASE}-$enhancement.jpg" | awk '{print $1}')
+    if [ $filesize > 20480 ]; then
+     if [ -f "${IMAGE_FILE_BASE}-$enhancement.jpg" ]; then
```

Comment:
Note to self: This likely belongs before the filesize assignment and check vs. after (validate the file exists, then check the size - attempting the other way around might be problematic).